include/nbl/ext/MitsubaLoader/ParserUtil.h

```diff
@@ -44,7 +48,7 @@ class ElementPool // : public std::tuple<core::vector<types>...>
 {
 		core::SimpleBlockBasedAllocator<core::LinearAddressAllocator<uint32_t>,core::aligned_allocator> poolAllocator;
 	public:
-		ElementPool() : poolAllocator(4096u*1024u, 256u) {}
+		ElementPool() : poolAllocator(4096u*1024u, 256u, 256u) {} // TODO: is it correct?
```

Comment:
no, 4096u*1024u, 1u, 256u 

the new parameter (min block count) comes inbetween